src/components/ChartContainer/index.js

```diff
@@ -95,6 +92,9 @@ class ChartContainer extends Component {
       .domain(subDomain)
       .range([0, chartWidth]);
     const children = React.Children.map(this.props.children, child => {
+      if (!child) {
```

Comment:
Can this ever be the case?

Reply:
Yes. When I remove the child dynamically, this.props.children[1] === false.

I have no idea why.